templates/frontend/components/languageSwitcher.tpl

```diff
@@ -12,14 +12,14 @@
  * @uses $id string A unique ID for this language toggle
  *}
 
-<ul id="{$id|escape}" class="language-toggle nav nav-tabs">
+<ul id="{$id|escape}" class="nav nav-tabs">
 	<li class="nav-item dropdown">
-		<a class="main-header__lang-link dropdown-toggle" id="languageToggleMenu{$id|escape}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+		<a class="dropdown-toggle main-header__lang-link" id="languageToggleMenu{$id|escape}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
 			<span class="sr-only">{translate key="plugins.themes.immersion.language.toggle"}</span>
 			{$languageToggleLocales[$currentLocale]|escape}
 		</a>
 
-		<ul class="dropdown-menu dropdown-menu-left" aria-labelledby="languageToggleMenu{$id|escape}">
+		<ul class="dropdown-menu" aria-labelledby="languageToggleMenu{$id|escape}">
```

Comment:
@sssoz, the class that is removed here has associated styles that come from Bootstrap: .dropdown-menu-left{right:auto;left:0}. Have you considered it for alignment of items? In any case, that menu (language toggle item) requires additional styling.

Reply:
It already aligns itself on the left by default -- `dropdown-menu-left` doesn’t seem to be found in Bootstrap 4’s components

Reply:
Also, yes I am trying to rely less on Bootstrap’s menu styling, since it doesn’t work with most of the themes anyway -- It’s mostly used for its grid / container system and a few JS-based widgets like modals!

Reply:
You can put this class to search here: https://getbootstrap.com/docs/4.3/dist/css/bootstrap.min.css
I don't remember the actual reason, it's related to some styling issue. That's OK if you are not planning to use it, just remember that all dropdowns and, especially, language toggle need additional styling management.